Update/toki_001.txt

```diff
@@ -172,7 +172,7 @@ void main()
 //rというのも、今彼女が向き合っている『その人』――姉の巴の身なりと様子があまりにも『すさまじかった』からだ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "というのも、今彼女が向き合っている『その人』——姉の巴の身なりと様子があまりにも『すさまじかった』からだ。",
-			NULL, "Because once she was standing face to face with the person she was waiting for ...her sister Tomoe's dress and appearance looked positively <i>frightening</i>.\"", Line_Normal);
+			NULL, "Because once she was standing face to face with the person she was waiting for ...her sister Tomoe's dress and appearance looked positively <i>frightening</i>.\", Line_Normal);
```

Comment:
```suggestion
			NULL, "Because once she was standing face to face with the person she was waiting for ...her sister Tomoe's dress and appearance looked positively <i>frightening</i>.", Line_Normal);
```